hal/armv7m/imxrt/10xx/imxrt.c

```diff
@@ -1689,6 +1691,12 @@ int _imxrt_gpioGetPort(unsigned int d, u32 *val)
 /* MPU */
 
 
+u32 _imxrt_typeMPU(void)
+{
+	return *(imxrt_common.mpu + mpu_type);
+}
+
+
```

Comment:
What is the reason of defining function which substitutes single line of simple code?